androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/screens/arrival/hazardassesment/HazardAssessment.kt

```diff
@@ -92,4 +125,48 @@ fun HazardAssessment(
             }
         }
     }
+}
+
+@Composable
+fun Laws(
```

Comment:
I'm not very fond of these named wrapper composables, and I think we should focus on creating _resusable_ composables instead. 
And in cases where we want to create single-use composables, I think that they should aim to reduce the amount of boilerplate code. e.g. Does the `Laws`-composable ever have any other input than `laws.keys.toTypedArray()` and other labels than `laws`)